packages/babel-generator/src/index.js

```diff
@@ -111,7 +111,10 @@ export class CodeGenerator {
   }
 }
 
-export default function (ast: Object, opts: Object, code: string): Object {
+function generator(ast: Object, opts: Object, code: string): Object {
   const gen = new Generator(ast, opts, code);
   return gen.generate();
 }
+
+export default generator;
+export { generator };
```

Comment:
`export { parse as default }` is required to create a live binding:
```suggestion
export { generator as default };
```

Using `export default <expression>` creates a snapshot of the `expression`.